### ISSUE TITLE
HTTP code 404 "Invalid URL, sorry", lack of details

### ISSUE THREAD

**Issue by thican** [kd2org/picodav#3]
When I try to reach the `index.php` inside my dedicated subdirecoty for Picodav, I receive the answer "Invalid URL, sorry" and a 404 HTTP code in my web browser.

It looks like related to this code, currently at line 1974:
```
    if (!$dav->route($uri)) {
        http_response_code(404);
        die('Invalid URL, sorry');
    }
```

I have no idea what’s wrong, the ownership and right on files looks correct from my point of view once the setup is done:
```
% ls -la /srv/www/example.com/path/to/picodav/
total 96K
drwxr-s--- 4 thican www-data   82 Dec 25 20:46 ./
drwxr-s--- 7 thican www-data 4.0K Dec 25 20:44 ../
drwxrws--- 2 thican www-data    6 Dec 25 19:46 dir1/
drwxrws--- 2 thican www-data    6 Dec 25 19:47 dir2/
-rw-r----- 1 thican www-data  410 Dec 25 20:46 .htaccess
-rw-r----- 1 thican www-data  84K Dec 25 20:02 index.php
-rw-r----- 1 thican www-data  261 Dec 25 19:51 .picodav.ini
```

**Comment by bohwaz** [comment]
Can you please try with latest release? This means that the URL you provided is not handled by PicoDAV. Please provide which URI you are trying to fetch.

**Comment by thican** [comment]
Hello, thanks for your fix in commit d1dd1e5778a566219ecd2a1c463eed837d56eb89, it works.
The URI was simply the same than provided in example for issue #4, i.e `https://example.com/path/to/picodav/` or `https://example.com/path/to/picodav/index.php`.

I think we can close the issue, maybe just the message line 1994 (previously 1974 as above) could get an improvement.

**Comment by bohwaz** [comment]
Thanks, the message has been changed.
